cmd/av/stack_sync.go

```diff
@@ -122,13 +114,32 @@ base branch.
 			if state.CurrentBranch == "" {
 				return errors.New("no sync in progress")
 			}
+
+			// Abort the rebase if we need to
```

Comment:
so previously the abort was a no-op? We were only clearing the state from the refs without actually updating any references?

Reply:
Yep, that's correct. 🙃 Kind of an obvious oversight in retrospect but hindsight is 20/20 😅